src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/HashFunctions.cs

```diff
@@ -2,18 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
```

Comment:
This file only contains value types, so there is no extra work needed to remove `#nullable disable`.